packages/mds-logger/index.ts

```diff
@@ -186,29 +188,25 @@ function info(...msg: any) {
 }
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-async function warn(...msg: any) {
-  try {
-    if (env.QUIET) {
-      return
-    }
+async function warn(...msg: any[]): Promise<any[]> {
```

Comment:
Not required for this PR, as it probably would increase the footprint of this change, but I would be inclined to use `unknown` instead of `any` in these sorts of functions.
>“TypeScript 3.0 introduces a new type called unknown that does exactly that. Much like any, any value is assignable to unknown; however, unlike any, unknown is assignable to almost nothing else without a type assertion. You also can’t access any properties off of an unknown, nor can you call/construct them.”